crud.py

```diff
@@ -18,22 +18,22 @@ def create_manager(db: Session, manager: schemas.ManagerCreate):
     db.refresh(db_manager)
     return db_manager
 
-def get_manager_by_manager_name(db: Session, manager_firstname: str, manager_lastname: str):
-    db_manager = db.query(models.Manager).filter(and_(models.Manager.manager_firstname == manager_firstname, models.Manager.manager_lastname == manager_lastname)).one()
+def get_manager_by_manager_username(db: Session, manager_username: str):
+    db_manager = db.query(models.Manager).filter(models.Manager.manager_firstname == manager_username).one()
```

Comment:
change `manager_firstname` to `manager_username` on the left of the `==`

Reply:
Ok, my mistake.